README.md

```diff
@@ -1 +1,64 @@
-# nvim-package-specification
\ No newline at end of file
+# `plugin.json`
+
+The neovim package specification supports a single, top-level package metadata file. This file can be *either* 'plugin.lua' or 'plugin.json'. The file is a superset of the Rockspec File Format and can contain the following fields
+
+* `package` : String, the name of the package
+
+* `version` : String, the version of the package. Should obey semantic versioning conventions and be prefixed with a `v`, for example, `v0.1.0`
+
+* `specification_version` : String, the current specification version. (v0.1.0) at this time.
+
+
+* `source` : Table, the source is a table that contains a `url` field, which points to either the git commit of the current version of the package, or the source tarball.
+
+* `description` : Table, the description is a table that includes the following nested fields:
+	* `summary` : String, a short description of the package, typically less than 100 character long.
+	* `detailed` : String, a long-form description of the package, this should convey the package's principal functionality to the user without being as detailed as the package readme.
+	* `homepage` : This is the homepage of the package, which in most cases will be the GitHub URL.
+	* `License` : This is the License type of the package.
```

Comment:
Should this be a SPDX license identifier, or just an arbitrary string?

Reply:
Yeah, good idea to put the link there :)